sdk/react/widgets/controlBarWidgets.js

```diff
@@ -258,11 +259,18 @@ class controlBarWidget extends React.Component {
 
   playbackSpeedWidget = (options) => {
     let widget = null;
+    
+    // Create accessibility label for selected playback speed rate button
+    
+    const playbackSpeedRateWithoutFostfix = options.selectedPlaybackSpeedRate.slice(0,-1);
```

Comment:
There is a typo here: playbackSpeedRateWithoutFostfix -> playbackSpeedRateWithoutPostfix
Should be PostFix not FostFix